app/controllers/coronavirus_form/offer_other_support_controller.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+class CoronavirusForm::OfferOtherSupportController < ApplicationController
+  include ActionView::Helpers::SanitizeHelper
+  include FieldValidationHelper
+
+  def show
+    render "coronavirus_form/#{PAGE}"
+  end
+
+  def submit
+    support_other = sanitize(params[:support_other]).presence
+
+    session[:offer_other_support] = support_other
+
+    invalid_fields = validate_radio_field(
+      PAGE,
+      other: support_other,
+    )
+
+    if invalid_fields.any?
+      flash.now[:validation] = invalid_fields
+      render "coronavirus_form/#{PAGE}"
+    elsif session["check_answers_seen"]
+      redirect_to controller: "coronavirus_form/check_answers", action: "show"
+    elsif session[:medical_equipment] == I18n.t("coronavirus_form.medical_equipment.options.option_yes.label")
```

Comment:
Does `medical_equipment` need to be `support_other` or could the line be removed?

Reply:
yeah sorry, just put this up to have somewhere to park it, still [WIP]!